src/config/anvil/config.ts

```diff
@@ -26,7 +26,7 @@ const config: SiteConfig = {
         },
         {
           label: "Datasets",
-          url: `${getDatasetsEnvironmentUrl()}data`,
+          url: "/explore/datasets",
```

Comment:
@hannes-ucsc can you please revert this change? Thanks! D

Reply:
I anticipated this change being controversial. I didn't understand `getDatasetsEnvironmentUrl` so I just clobbered it with the hard-coded line. Is there any way this could be made dynamic so that we don't have to maintain a branch for this one-line difference? We have a way to inject environment variables into the build. See my next comment.

Reply:
We are in the process of implementing the catalog/explorer selector now, so how about for right now, we make this PR deal with the prod build and deploy only, and we remove the setting of the value for the "Datasets" link as this will change soon anyhow.

We want to keep the catalog/explorer selector on its own branch until we are ready to deploy this to anvilproject.org, which I believe is in several weeks at the earliest. We will need to do deployments until the switchover, so I don't want to make changes to `main` that change the functionality for the managed access deployment only.

If I have the schedule wrong, let me know.
cc: @bvizzier-ucsc